.travis.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     - stage: build
       name: 'Unminified Build'
       script:
-        - unbuffer node build-system/pr-check/unminified-build.js
+        - travis wait 30 unbuffer node build-system/pr-check/unminified-build.js
```

Comment:
Without this, the longer `--coverage` build will timeout on Travis

Reply:
As written, this will increase the time taken by all Travis builds because the "test" stage will now have to wait for a 30 minute "build" stage.

The good news is that we should be able to solve this problem with the move to CircleCI, which I believe allows for more sophisticated matrices (e.g. create a separate independent "coverage" stage, which starts at the beginning and runs in parallel with the "build" and "test" stages, which run serially.)

I expect to make progress on this in the next week or so, will loop you in when the coast is clear to try this out.

Reply:
Thanks for reminding me about this, forgot to note it! My thinking was this would only be viable for something like the Nightly builds, and probably shouldn't be a part of CI during the development day. Even if Circle CI supports splitting the work off, e2e test coverage shouldn't fluctuate massively from one commit to another, and daily data would be sufficient as a metric.